tools/sitemap/create.js

```diff
@@ -0,0 +1,63 @@
+import fs from 'fs/promises';
+import convert from 'xml-js';
+import path from 'path';
+
+const QUERY_INDEX_URL = 'https://www.bitdefender.com.au/solutions/query-index.json';
```

Comment:
this needs to be changed in order to generate the corect sitemap.xml, witch you can test it on http://localhost:3000/_src/sitemap.xml

Reply:
@vradulescu-bd I have modified the sitemap, it now generates the correct sitemap. Only problem is that this is only for one of the two websites, namely zh-hk